maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/NodeHelper.java

```diff
@@ -0,0 +1,73 @@
+package org.openmrs.maven.plugins.utility;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.maven.execution.MavenSession;
+import org.apache.maven.plugin.BuildPluginManager;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.project.MavenProject;
+import org.twdata.maven.mojoexecutor.MojoExecutor;
+import org.w3c.dom.Node;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.twdata.maven.mojoexecutor.MojoExecutor.artifactId;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.configuration;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.element;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.executeMojo;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.executionEnvironment;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.goal;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.groupId;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.plugin;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.version;
+
+public class NodeHelper {
+    private static final String FRONTEND_MAVEN_GROUP_ID = "com.github.eirslett";
+    private static final String FRONTEND_MAVEN_ARTIFACT_ID = "frontend-maven-plugin";
+    private static final String FRONTEND_MAVEN_VERSION = "1.12.0";
```

Comment:
Probably best to stick with 1.11.3, if possible. 1.12.0 requires Maven 3.6.0+, which we actually don't have on the CI build agents (IIRC, we tried to upgrade to Maven 3.6 and it broke some builds).